lighthouse-core/lib/network-recorder.js

```diff
@@ -107,9 +107,15 @@ class NetworkRecorder extends EventEmitter {
         return;
       }
 
+      // QUIC network requests don't always "finish" even when they're done loading data
+      // Use receivedHeaders instead, see https://github.com/GoogleChrome/lighthouse/issues/5254
+      const isQUIC = record._responseHeaders && record._responseHeaders
```

Comment:
would be nice pulled out into a function to separate determining if request is QUIC vs what to do with QUIC requests

Reply:
it could also be a function that's like `isQuicAndFinished(record)` to absorb logic below, too :)

Reply:
sure